python/TestHarness/schedulers/Job.py

```diff
@@ -764,9 +764,7 @@ def storeResults(self, scheduler):
         joint_status = self.getJointStatus()
 
         # Base job data
-        job_data = {'name'                 : self.getTestNameShort(),
-                    'long_name'            : self.getTestName(),
```

Comment:
I trust we are not using these anymore?

Reply:
Those were actually added in my last PR: they're unused now because they're contained in the keys for the main map